modules/pam_motd/pam_motd.8.xml

```diff
@@ -132,11 +132,27 @@
   <refsect1 id='pam_motd-return_values'>
     <title>RETURN VALUES</title>
     <variablelist>
+      <varlistentry>
+        <term>PAM_ABORT</term>
+        <listitem>
+           <para>
+             Not all relevant data or options could be gotten.
```

Comment:
got or obtained - although sometimes used gotten is weird.

Reply:
I agree. I copied this over from `pam_env` and the same phrasing is present in other modules, too, thus I did not want to diverge.